test/lib/usd/translators/testUsdExportUVTransforms.py

```diff
@@ -0,0 +1,97 @@
+#!/pxrpythonsubst
+#
+# Copyright 2020 Apple Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import unittest
+
+import fixturesUtils
+from maya import cmds
+from maya import standalone
+from pxr import Usd
+from pxr import UsdShade
+
+
+def findTransformPrim(root):
+    for prim in root.GetAllChildren():
+        if not prim.IsA(UsdShade.Shader):
+            continue
+
+        shader = UsdShade.Shader(prim)
+        shader_type = shader.GetIdAttr().Get()
+        if shader_type == "UsdTransform2d":
+            return prim
+
+        transform = findTransformPrim(prim)
+        if transform:
+            return transform
+
+
+class testUsdExportUVTransforms(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.temp_dir = fixturesUtils.setUpClass(__file__, initializeStandalone=False)
```

Comment:
I think we actually **do** want the `standalone.initialize()` call that this would otherwise do, so I think remove the `initializeStandlone=False` here?

Reply:
Ah my bad. I write the tests interactively, so had disabled that and forgotten to remove the statements. Same for the return below. I'll remove those.